### PR TITLE
Delete old `d2l-rubric-criteria-group-mobile` component and combine with criteria mobile - US132977

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fuserstory%2F612449867053

### CODE REVIEW

**Review by devpow112 — APPROVED** [review]

File: src/components/rubric-criteria-group-mobile.js [hunk 1]
```diff
@@ -6,41 +6,52 @@ import { AssessmentMixin } from '../mixins/assessment-mixin.js';
...
+import { heading3Styles } from '@brightspace-ui/core/components/typography/styles.js';
 import { LocalizeMixin } from '../mixins/localize-mixin.js';
 import { OptimisticUpdateDispatcher } from '../helpers/optimistic-update-dispatcher.js';
 import { RubricCriteriaEntity } from '../entities/rubric-criteria-entity.js';
+import { RubricCriteriaGroupEntity } from '../entities/rubric-criteria-group-entity.js';
 import { rubricStyles } from '../styles/rubric-styles.js';
+import { TelemetryMixin } from '../mixins/telemetry-mixin.js';
 
-class RubricCritiaMobile extends composeMixins(
+class RubricCritiaGroupMobile extends composeMixins(
```
Comment (imcdonald-d2l) [inline src/components/rubric-criteria-group-mobile.js:18]: Re-named this file/component to `rubric-criteria-group-mobile` to accurately represent what this component renders.

**Comment by devpow112** [comment]
> Visual diff tests failed - pull request #1461 has been opened with the updated goldens.

For some reason all the images are a little bit taller. I don't see any other difference.

**Review by MitchelPaulin — DISMISSED** [review]

File: src/components/rubric-criteria-group-mobile.js [hunk 2]
```diff
@@ -89,6 +110,7 @@ class RubricCritiaMobile extends composeMixins(
 
 	render() {
 		return html`
+			<h3>${this._groupName}</h3>
```
Comment (MitchelPaulin) [inline src/components/rubric-criteria-group-mobile.js:113]: Do you need to add the rubric heading styles class here? Might be why vdiff is failing

Comment (imcdonald-d2l) [reply src/components/rubric-criteria-group-mobile.js:113]: I didn't think it was necessary but I'll give it a try 👍🏻

Comment (MitchelPaulin) [reply src/components/rubric-criteria-group-mobile.js:113]: If not I'm ok with just updating the goldens

**Review by MitchelPaulin — APPROVED** [review]

File: src/components/rubric-criteria-group-mobile.js (hunk 2 above)
Comment (imcdonald-d2l) [reply src/components/rubric-criteria-group-mobile.js]: Seems to have fixed it 🎉

**Pull request merged**